### ISSUE TITLE
VideoPress: Show video quick actions tooltip on focus

### ISSUE THREAD

**Issue by dhasilva** [Automattic/jetpack#27332]
The tooltips are only shown on hover.

p1668009383019279/1668003224.122799-slack-C03TA48NSUX
